core/db/entities.py

```diff
@@ -124,6 +124,139 @@ class ProjectProgress(db.Model):
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="capital_state"), nullable=False)
+    amount = sqla.Column(sqla.Float(), nullable=False)
+
+    # TODO: does this unique index look right?
+    # Unique index for data integrity. There can't be multiple capital data rows for a single project with
+    # the same date range and capital metrics.
+    __table_args__ = (
+        sqla.Index(
+            "ix_unique_capital",
+            "project_id",
+            "start_date",
+            "end_date",
+            "state",
+            unique=True,
+        ),
+    )
+
+
+class IndirectFundSecured(db.Model):
+    """Stores Indirect Fund Secured Data for Projects."""
+
+    __tablename__ = "indirect_fund_secured_data"
+
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
+    project_id = sqla.Column(sqla.String(), sqla.ForeignKey("project_dim.id"), nullable=False)
+    start_date = sqla.Column(sqla.DateTime(), nullable=False)
+    end_date = sqla.Column(sqla.DateTime(), nullable=False)
+
+    # TODO: Should this reference entities in organisation model, or is it free-text
+    funding_source_name = sqla.Column(sqla.String(), nullable=False)
+    funding_source_category = sqla.Column(
+        sqla.Enum(const.FundingSourceCategoryEnum, name="indirect_secured_fund_source_category"),
+        nullable=False,
+    )
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="indirect_secured_fund_state"), nullable=False)
+    amount = sqla.Column(sqla.Float(), nullable=False)
+
+    # TODO: does this unique index look right?
+    # Unique index for data integrity. There can't be multiple indirect fund secured rows for a single project with
+    # the same date range and fund metrics.
+    __table_args__ = (
+        sqla.Index(
+            "ix_unique_indirect_fund_secured",
+            "project_id",
+            "start_date",
+            "end_date",
+            "funding_source_name",
+            "state",
+            unique=True,
+        ),
+    )
+
+
+class IndirectFundUnsecured(db.Model):
+    """Stores Indirect Fund Unsecured Data for Projects."""
+
+    __tablename__ = "indirect_fund_unsecured_data"
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
+    project_id = sqla.Column(sqla.String(), sqla.ForeignKey("project_dim.id"), nullable=False)
+    start_date = sqla.Column(sqla.DateTime(), nullable=False)
+    end_date = sqla.Column(sqla.DateTime(), nullable=False)
+
+    # TODO: Should this reference entities in organisation model, or is it free-text
+    funding_source_name = sqla.Column(sqla.String(), nullable=False)
+    funding_source_category = sqla.Column(
+        sqla.Enum(const.FundingSourceCategoryEnum, name="indirect_unsecured_fund_source_category"),
+        nullable=False,
+    )
+    state = sqla.Column(sqla.Enum(const.StateEnum, name="indirect_unsecured_fund_state"), nullable=False)
```

Comment:
`"indirect_unsecured_fund_source_category"` should be `""indirect_fund_unsecured_source_category""`
`"indirect_unsecured_fund_state"` should be `"indirect_fund_unsecured_state"`